test/Puzzle-test.js

```diff
@@ -3,11 +3,20 @@ const expect = chai.expect;
 
 import Puzzle from '../src/classes/Puzzle'
 
-describe('Puzzle', function() {
+describe.only('Puzzle', function() {
```

Comment:
also if anyone goes to do more testing I have a .only() method on the describe block so any tests outside of Puzzle-test.js will not appear in console after running ```npm test```.